definitions/json/CaseEventToFields.json

```diff
@@ -2745,6 +2745,16 @@
     "PageFieldDisplayOrder": 6,
     "ShowSummaryChangeOption": "Y"
   },
+  {
+    "CaseTypeID": "ET_EnglandWales",
+    "CaseEventID": "et1Vetting",
+    "CaseFieldID": "trackType",
+    "FieldShowCondition": "et1JurisdictionCodeGeneralNotes=\"dummy\"",
```

Comment:
Not the best of fieldShow conditions - what if someone actually typed in dummy on the general notes box? This field would then show